addons/packages/pinniped/0.12.0/bundle/config/upstream/_ytt_lib/supervisor/values.yaml

```diff
@@ -73,3 +73,6 @@ api_group_suffix: pinniped.dev
 #! Optional.
 https_proxy: #! e.g. http://proxy.example.com
 no_proxy: "$(KUBERNETES_SERVICE_HOST),169.254.169.254,127.0.0.1,localhost,.svc,.cluster.local" #! do not proxy Kubernetes endpoints
+
+#! Set the defalut Pinniped TLS secret name on the Supervisor containers.
+tls_secret_name: pinniped-supervisor-default-tls-certificate
```

Comment:
`nit` on the newline at the end of the file, otherwise, nice!

Reply:
ooh just realized i misspelled default lol

Reply:
Just fixed both.

Reply:
Though GitHub isn't displaying the newline like I'd hoped.